### ISSUE TITLE
nothing works

### ISSUE THREAD

**Issue by allmors** [Leeon123/CC-attack#77]
The newly upgraded code doesn't seem to have any effect. I tested my website without any protection and found that it didn't work at all.
i use:`python3 cc.py -url https://www.exmple.com/?s=d -m cc -v 5 -s 100 -t 500 -down -f d.txt -b 1`

**Comment by Leeon123** [comment]
Try multiproc.sh and socks4, socks5 proxies in public are too less

**Comment by allmors** [comment]
> Try multiproc.sh and socks4, socks5 proxies in public are too less

I tested it and it seems that it has nothing to do with the proxy type, it is related to the latest update. The version before adding the http proxy is very effective. The latest version of adding the http proxy seems to have no effect. I backed up the code in 2020. , that set of code still has attack effect, but after adding http proxy recently, it has no effect
https://i0.hdslb.com/bfs/album/c81bbc619a6abd3337187e4fdc73516d2ea882ef.png (2020)
